pkg/cmd/codespace/create.go

```diff
@@ -35,6 +39,7 @@ func newCreateCmd(app *App) *cobra.Command {
 	createCmd.Flags().StringVarP(&opts.repo, "repo", "r", "", "repository name with owner: user/repo")
 	createCmd.Flags().StringVarP(&opts.branch, "branch", "b", "", "repository branch")
 	createCmd.Flags().StringVarP(&opts.machine, "machine", "m", "", "hardware specifications for the VM")
+	createCmd.Flags().BoolVarP(&opts.permissionsOptOut, "skip-permissions", "", false, "do not accept additional permissions requested by the codespace")
```

Comment:
I would love feedback on this new flag. I'm not sure `skip-permissions` is the best name since it sounds like we are somehow skipping all permissions checks which we are not, we are simply not prompting them for the additional permissions and continuing on as if they hit 'cancel' in the UI.

Perhaps something like `decline-permissions` instead? cc @reybard @gdomingu @asciimike

Reply:
is `opt-out-devcontainer-permissions` too long?

Reply:
That does seem a bit long winded

Reply:
It's equivalent to the client to clicking the "Continue without authorizing" button in the UI so what about drawing from that and naming it something like `--continue-without-authorizing`?

Reply:
I like ☝️ or fine staying with `skip-permissions` the others seem too long.

Reply:
@mislav suggested `--default-permissions` which I like as well

Reply:
Actually, now that I think about it, this flag is essentially just saying 'dont prompt me'. So perhaps `--no-prompt` or `--no-permissions-prompt` as suggested is better